### PR TITLE
Feature/travisci internal bundle

### PR DESCRIPTION
**Pull request by travissluka**
## Description

The "bundle" `CMakeLists.txt` was moved internally into the `bundle/` directory. This mirrors what was done for soca-cice6. Now soca-cice6 and soca can each have  their own `release/stable-nightly` branch contained in their own repository. Slightly cleaner that way. soca-bundle will be reserved for coupled stuff.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,5 +8,20 @@ JEDI encapsulation of MOM6
...
 This software is licensed under the terms of the Apache Licence Version 2.0 which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
 
+See the [soca ReadTheDocs](https://jointcenterforsatellitedataassimilation-soca.readthedocs-hosted.com/en/latest/?badge=latest) page for more documentation.
 
-See the [soca-bundle](https://github.com/JCSDA/soca-bundle) and the [soca ReadTheDocs](https://jointcenterforsatellitedataassimilation-soca.readthedocs-hosted.com/en/latest/?badge=latest) page for more documentation.
\ No newline at end of file
+## Building
+
+1. If building as part of the larger SOCA coupled system, see the [soca-bundle](https://github.com/JCSDA/soca-bundle)
+
```
Comment (guillaumevernieres) [inline README.md:16]: :+1:  Nicely worded. Time to revive the renaming github issue, I think @aerorahul did that work a while back. I forgot why we didn't merge ...

File: README.md [hunk 2]
```diff
@@ -8,5 +8,20 @@ JEDI encapsulation of MOM6
...
+2. Otherwise, for just the MOM6 SOCA component: unlike other JEDI projects, this project contains its own bundle, it can be built as follows
+```
+git clone https://github.com/JCSDA/soca.git
+mkdir -p soca_build
+cd soca_build
+ecbuild ../soca/bundle
+cd soca
```
Comment (guillaumevernieres) [inline README.md:23]: Beside brainwashing our users (which I'm all for) to do more soca stuff, is there an other use for that step :point_up:  ?

Comment (travissluka) [reply README.md:23]: the `cd soca` part? It builds faster, it won't build all upstream executables, like the oops toy models

**Review by guillaumevernieres — APPROVED** [review]
I'm blind to typos, but I tested the build. All good.

**Pull request merged**